src/components/top-bar/top-bar.js

```diff
@@ -0,0 +1,61 @@
+// todo: use ES6 instead of namespace
+(function (window, document) {
+	'use strict';
+
+	window.SG = window.SG || {};
```

Comment:
This kind of boilerplate is not necessary. Let's just write a namespacing function and make a core styleguide js file for common functions/utility like this.